system/disk_usage.tf

```diff
@@ -0,0 +1,45 @@
+resource "datadog_monitor" "fs_free_inodes" {
+  name    = "Insufficient of count free inodes ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "Insufficient of count free inodes ${var.free_inodes_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
+  query   = "avg(last_${var.free_inodes_time}):avg:system.fs.inodes.free{host:${data.aws_instance.monitored.instance_id}} by {device,host} < ${var.free_inodes_critical_state_value}"
+
+  thresholds {
+    ok       = "${var.free_inodes_ok_state_value}"
+    warning  = "${var.free_inodes_warning_state_value}"
+    critical = "${var.free_inodes_critical_state_value}"
+  }
+
+  renotify_interval = "${var.renotify_interval_mins}"
+  new_host_delay    = "${var.new_host_delay}"
+  notify_no_data    = "${var.notify_no_data}"
+
+  silenced {
+    "*" = "${var.active}"
+  }
+
+  tags = ["${module.label.tags}"]
+}
+
+resource "datadog_monitor" "disk_freespace" {
+  name    = "Insufficient of free disk space ${module.label.id}"
+  type    = "${var.alert_type}"
+  message = "Insufficient of free disk space ${var.freespace_bytes_time} on host: ${data.aws_instance.monitored.instance_id} with IP: ${data.aws_instance.monitored.instance_id.public_ip}"
```

Comment:
Disk space is insufficient for `${var.freespace_bytes_time}` on host: ...

Check the other files too.